src/modules/auth/index.js

```diff
@@ -97,6 +98,7 @@ export default class AuthModule extends Module {
 			}
 		}
 		this.App.removeSplashScreenRequestor(this);
+		this.notifyOnExpiredSession(this);
```

Comment:
This will call update user info again at the start. That's incorrect b/c userinfo will be called anyway.